src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx

```diff
@@ -9,8 +9,8 @@ import { KeyIcon } from "../Icons/KeyIcon/KeyIcon";
 import { LockIcon } from "../Icons/LockIcon/LockIcon";
 import { NavBarCrossIcon } from "../Icons/NavBarCrossIcon/NavBarCrossIcon";
 import { RightChevron } from "../Icons/RightChevron/RightChevron";
-import { SwitchIcon } from "../Icons/SwitchIcon/SwitchIcon";
-import { WorldIcon } from "../Icons/WorldIcon/WorldIcon";
+import { WhiteSwitchIcon } from "../Icons/WhiteSwitchIcon/WhiteSwitchIcon";
+import { WhiteWorldIcon } from "../Icons/WhiteWorldIcon/WhiteWorldIcon";
```

Comment:
What do you think about having `Icons/WorldIcon/(White|Black)WorldIcon`? The repo has already quite a lot of folder :).

Reply:
OK fixed ab546d4 I did the same for `SwitchIcon`